tools/SourceKit/lib/SwiftLang/SwiftCompletion.cpp

```diff
@@ -75,9 +74,6 @@ struct SwiftToSourceKitCompletionAdapter {
                            bool legacyLiteralToKeyword,
                            bool annotatedDescription);
 
-  static void getResultSourceText(const CodeCompletionString *CCStr,
-                                  raw_ostream &OS);
-
   static void getResultAssociatedUSRs(ArrayRef<StringRef> AssocUSRs,
                                       raw_ostream &OS);
```

Comment:
Do you think it's not worth to move this function to `libIDE` as well?

Reply:
In my opinion this one is more sourcekitd-specific, since it's specifically formatting those strings as a space-separated string.